tests/model_serving/model_runtime/vllm/quantization/test_openhermes-2_5_mistral-7b_awq.py

```diff
@@ -0,0 +1,208 @@
+import pytest
+from simple_logger.logger import get_logger
+from utilities.constants import KServeDeploymentType
+from tests.model_serving.model_runtime.vllm.utils import fetch_openai_response, run_raw_inference
+from tests.model_serving.model_runtime.vllm.constant import VLLM_SUPPORTED_QUANTIZATION
+
+LOGGER = get_logger(name=__name__)
+
+
+serving_argument = [
+    "--model=/mnt/models",
+    "--uvicorn-log-level=debug",
+    "--chat-template=/app/data/template/tool_chat_template_mistral.jinja",
+]
+
+model_path = "TheBloke/OpenHermes-2.5-Mistral-7B-AWQ"
+
+pytestmark = pytest.mark.usefixtures("skip_if_no_supported_accelerator_type", "valid_aws_config")
+
+
+@pytest.mark.parametrize(
+    "model_namespace, s3_models_storage_uri, serving_runtime, vllm_inference_service",
+    [
+        pytest.param(
```

Comment:
i suggest adding `id` to pytest.param; this way it is easier to run a single test from all parameterized tests

Reply:
done